### ISSUE TITLE
Limit Minute selection for From & To

### ISSUE THREAD

**Issue by christian-lo** [kimai/kimai#4093]
### Describe the problem

There are a lot of options for minute selection starting from 12 a.m. but my working time usually starts at 8 a.m..

Would be nice to have these time range adjusted to the calendar view, Start / End of visual time range for example. 

### Describe the solution you'd like

Adjust the time range to the calendar view, Start / End of visual time range for example. 

### Describe alternatives you've considered

-

### Screenshots
<img width="737" alt="Bildschirmfoto 2023-06-08 um 12 25 39" src="https://github.com/kimai/kimai/assets/81588467/5283e716-ddeb-4743-b35e-995c20a2c2df">

<img width="996" alt="Bildschirmfoto 2023-06-08 um 12 26 17" src="https://github.com/kimai/kimai/assets/81588467/b1c13016-a7d0-431c-8dd5-43954fdb8245">

**Comment by kevinpapst** [comment]
You mean this, right: https://github.com/kimai/kimai/issues/3881 ?

**Comment by christian-lo** [comment]
not really, I mean to limit the available options to the times set in Settings - Calendar - Start / End of visible time range.

Because you can always set the time manually, most selects will be within the usual working hours. If the view automatically scrolls to the time being to avoid scrolling when using the selection in the afternoon, will be handy. 

<img width="734" alt="Bildschirmfoto 2023-06-08 um 22 27 39" src="https://github.com/kimai/kimai/assets/81588467/f35eccbd-8280-46fa-aa43-68f156f5c44b">
